charts/opentelemetry-collector/values.yaml

```diff
@@ -83,6 +83,8 @@ configMap:
   # Specifies an existing ConfigMap to be mounted to the pod
   # The ConfigMap MUST include the collector configuration via a key named 'relay' or the collector will not start.
   existingName: ""
+  # Specifies relative path to the existing ConfigMap template file
```

Comment:
Let's explain that a user should set this value when they want the collector to change bc the configmap changed.